WeakAuras/Prototypes.lua

```diff
@@ -4493,6 +4493,7 @@ WeakAuras.event_prototypes = {
                 destName = UnitExists(destUnit) and UnitName(destUnit) or "",
                 show = true,
                 changed = true,
+                inverse = castType == "cast",
```

Comment:
Did you check that it now behaves like it did before?

Reply:
yes i tested cast and channeling with older version and it behaved as you said